airtable.go

```diff
@@ -59,6 +58,11 @@ func (t *GenericTable) List(opts Options) ([]Record, error) {
 		q.Add("maxRecords", fmt.Sprint(opts.MaxRecords))
 	}
 
+	// add offset
+	if opts.Offset != "" {
```

Comment:
Two things:

1. Any reason `Offset` is a string here? Is it because we are passing the next recordId?
2. I believe the `go way` of checking for empty string is: `len(opts.Offset) > 0`